ui/app/models/sync/destination.js

```diff
@@ -23,4 +24,27 @@ export default class SyncDestinationModel extends Model {
   get maskedParams() {
     return findDestination(this.type)?.maskedParams;
   }
+
+  @lazyCapabilities(apiPath`sys/sync/destinations/${'type'}/${'name'}`, 'type', 'name') destinationPath;
+  @lazyCapabilities(apiPath`sys/sync/destinations/${'type'}/${'name'}/associations/set`, 'type', 'name')
+  setAssociationPath;
+
+  get canCreate() {
+    return this.destinationPath.get('canCreate') !== false;
+  }
+  get canDelete() {
+    return this.destinationPath.get('canDelete') !== false;
+  }
+  get canEdit() {
+    return this.destinationPath.get('canUpdate') !== false;
+  }
+  get canRead() {
+    return this.destinationPath.get('canRead') !== false;
+  }
+  get canList() {
+    return this.destinationPath.get('canList') !== false;
```

Comment:
Listing will actually happen on the path without `name`? or is there a different list being accessed here? (just going off conventions for previous work, which I know this doesn't exactly match)

Reply:
Ahh yes good catch! I don't think we are actually going to be using that anyhow since the query is done in the model hook and the error is handled there so I'm going to remove this for now rather than adding another path.